phoenix-core/src/main/java/org/apache/phoenix/util/MetaDataUtil.java

```diff
@@ -678,13 +691,13 @@ public static SequenceKey getOldViewIndexSequenceKey(String tenantId, PName phys
         return new SequenceKey(isNamespaceMapped ? tenantId : null, schemaName, tableName, nSaltBuckets);
     }
 
-    public static String getViewIndexSequenceSchemaName(PName physicalName, boolean isNamespaceMapped) {
+    public static String getViewIndexSequenceSchemaName(PName logicalName, boolean isNamespaceMapped) {
```

Comment:
should be logicalParentName or logicalBaseTableName to make it clear that this is not the logical name of the view, but the suffix of the _IDX_ table so that all view indexes of the same base table get the same view index sequence.

Reply:
Will change to logicalBaseTableName